packages/trpc/server/routers/viewer/teams/get.handler.ts

```diff
@@ -37,15 +37,22 @@ export const getHandler = async ({ ctx, input }: GetOptions) => {
   // Hide Members of team when 1) Org is private and logged in user is not admin or owner
   // OR
   // 2)Team is private and logged in user is not admin or owner of team or Organization's admin or owner
-  const hideMembers =
-    (ctx.user.profile?.organization?.isPrivate && !ctx.user.organization?.isOrgAdmin) ||
-    (team.isPrivate &&
-      !(membership.role === MembershipRole.OWNER || membership.role === MembershipRole.ADMIN) &&
-      !ctx.user.organization?.isOrgAdmin);
+  function shouldHideMembers() {
+    const isOrgPrivate = ctx.user.profile?.organization?.isPrivate;
+    const isOrgAdminOrOwner = ctx.user.organization?.isOrgAdmin;
```

Comment:
For consistency, we should access the organization from same place.

Reply:
I think we should rename `isOrgAdmin` to `isOrgAdminOrOwner` only.

Reply:
I agree but i think we should handle this in another PR - I can create an issue for this